metrics/main.go

```diff
@@ -60,6 +60,8 @@ func main() {
 	// Add rbd mirror metrics collector to registry
 	collectors.RegisterRBDMirrorCollector(rbdRegistry, opts)
 
+	collectors.RegisterCephBlocklistCollector(rbdRegistry, opts)
```

Comment:
This isn't a rbd mirror specific metric. So it should be in customResourceRegistry.